www/manifest.json

```diff
@@ -1,19 +1,19 @@
-{

```

Comment:
also the commit log show the file as renamed without changes (https://github.com/BitBotFactory/poloniexlendingbot/pull/417/commits/ec485cc0f9ee86b4af3cf43c7bdf48db24074dc5)